napari/_vispy/vispy_scale_bar_visual.py

```diff
@@ -24,21 +28,22 @@ def __init__(self, viewer, parent=None, order=0):
             ]
         )
         self._default_color = np.array([1, 0, 1, 1])
-        self._target_length = 100
+        self._target_length = 150
         self._scale = 1
+        self._quantity = UNIT_REG("")  # unit-less
```

Comment:
Could we have maybe a constant/Emnum that was this empty string?

```python
NoUnit = ""  # or NO_UNIT = ""?
UNIT_REG(NoUnit)
```

Might be useful and more indicative moving forward?